src/components/Messages/Messages.js

```diff
@@ -24,14 +29,34 @@ const Messages = ( ): Node => {
     }
   );
 
+  const tabs = [
+    {
+      id: NOTIFICATIONS_ID,
+      text: "Notifications",
+      onClick: () => {
```

Comment:
This is really minor, but I prefer to use `onPress` or `onTap` since a user is not actually clicking a keyboard mouse on mobile.